.cspell.json

```diff
@@ -44,6 +44,11 @@
     "ukef",
     "UKRATEGRP",
     "venv",
+    "probabilityof",
```

Comment:
These all appear to be duplicated elsewhere in this list so can be removed.

Reply:
Good catch, it was a lot of duplicates, removed and sorted alphabetically.